test/epgsql_SUITE.erl

```diff
@@ -0,0 +1,115 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Main Test Suite for the epgsql module
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+-module(epgsql_SUITE).
+
+-export([ all/0
+        , init_per_suite/1
+        , end_per_suite/1
+        , init_per_testcase/2
+        , end_per_testcase/2
+        ]).
+
+-export([ equery_timeout/1
+        , execute_timeout/1
+        , prepared_query_timeout/1
+        , squery_timeout/1
+        ]).
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Includes
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/assert.hrl").
```

Comment:
It looks like `stdlib/include/assert.hrl` only available from Erlang 18, but our travis.yml includes even Erlang 15 currently.
AFAIK, it's ok to use `eunit/include/eunit.hrl` instead.